### ISSUE TITLE
VS Code will not build c programs with multiple .c source files 

### ISSUE THREAD

**Issue by Joeblack1788** [microsoft/vscode-docs#5228]
I have multiple C files in my program. task.json and launch.json were modified as recommend by various users in "stackoverflow" and VS installation. However, none are working for me. The following are changes that I made in "args":[]
1) "${fileDirname}/*.cpp",
2) Replace "${file}" with "${workspaceFolder}\\*.c"
3) Replace "${file}" with "${fileDirname}\\*.c"

I ran in the following commands in the command window and it worked
1) g++ ./*.c -o test
2) test

**Comment by gregvanl** [comment]
Quick question: are you using lowercase '.c' for the file extensions, rather than upper case '.C'.
This had caused issues for users in the past https://github.com/microsoft/vscode-docs/issues/5091

**Comment by gregvanl** [comment]
Closing as no response from user

**Comment by Joeblack1788** [comment]
Greg, 

Sorry for the late reply, I was moved to do other thing.
I test with the lower case "c". Still have the same issue.

Thanks

    On Friday, March 11, 2022, 06:57:08 PM CST, Greg Van Liew ***@***.***> wrote:  
 
 


Quick question: are you using lowercase '.c' for the file extensions, rather than upper case '.C'.
This had caused issues for users in the past #5091

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Joeblack1788** [comment]
The lower case c for file extension, it works when I used in the command window but not with VS code windows.

Thanks
    On Sunday, March 20, 2022, 12:58:08 PM CDT, Bill Sangchat ***@***.***> wrote:  
 
  Greg, 

Sorry for the late reply, I was moved to do other thing.
I test with the lower case "c". Still have the same issue.

Thanks

    On Friday, March 11, 2022, 06:57:08 PM CST, Greg Van Liew ***@***.***> wrote:  
 
 


Quick question: are you using lowercase '.c' for the file extensions, rather than upper case '.C'.
This had caused issues for users in the past #5091

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
